README.md

```diff
@@ -45,10 +45,16 @@ window, without the need to type in the full path of the binary.*
 
 ### Install `gows`
 
+First, tell go to download the package:
 ```
 go get -u github.com/bitrise-io/gows
 ```
 
+Next, tell go to install `gows`:
```

Comment:
`go install` should be enough by itself. It also downloads the repository just like `get`, but it also puts the binary in `$GOPATH/bin`.